test/e2e/create_cluster_test.go

```diff
@@ -1396,6 +1432,77 @@ func TestCreateClusterPrivateWithRouteKAS(t *testing.T) {
 	testCreateClusterPrivate(t, true)
 }
 
+// testCreateClusterWithDisabledCapabilities implements a test that creates a cluster
+// with the ImageRegistry capability disabled, then attempts to enable it.
+func testCreateClusterWithDisabledCapabilities(ctx context.Context, t *testing.T, g Gomega, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
```

Comment:
can we reuse TestCreateClusterCustomConfig for this validation and avoid loading the cloud account with a another HC?

Reply:
This function is being called from inside TestCreateClusterCustomConfig: https://github.com/openshift/hypershift/pull/5810/files#diff-1ab8a204348b5b2e9d5d5995903341902c62d82f84061b1b7cd9e0858702e6c1R1337
Or am I misunderstanding you?